src/config/datachart.json

```diff
@@ -498,22 +498,6 @@
                         "jp": "$.ig.DataSource からデータを要求するには、$.ig.DataSource により承諾されたリモート URL を指定します。"
                     }
                 },
-                "dataSourceType": {
-                    "type": "string",
-                    "defaultValue": null,
-                    "description": {
-                        "en": "Explicitly set data source type (such as \"json\"). Please refer to the documentation of $.ig.DataSource and its type property.",
-                        "jp": "データ ソースのタイプ (\"json\" など) を明示的に設定します。$.ig.DataSource とそのタイプ プロパティのドキュメントを参照してください。"
-                    }
-                },
-                "responseDataKey": {
```

Comment:
If the user setup uses remote datasource with remote url, is this property required?